packages/indexer-common/src/allocations/types.ts

```diff
@@ -175,3 +176,50 @@ export const parseGraphQLEpochs = (epoch: any): Epoch => ({
   totalIndexerRewards: epoch.totalIndexerRewards,
   totalDelegatorRewards: epoch.totalDelegatorRewards,
 })
+
+export interface NetworkEpochBlock {
+  network: string
+  epochNumber: number
+  startBlockNumber: number
+  startBlockHash: string
+  prevStartBlockNumber: number
+  prevStartBlockHash: string
+}
+
+export const NetworkAliases: { [key: string]: string } = {
+  mainnet: 'eip155:1',
+  rinkeby: 'eip155:4',
```

Comment:
The EBO doesn't currently track rinkeby. Is this a requirement?

Reply:
No, I will delete this:)